.pipelines/js-release.yml

```diff
@@ -5,7 +5,7 @@ trigger: none
 
 pool:
   name: Azure Pipelines
-  vmImage: ubuntu-20.04
+  vmImage: ubuntu-22.04
```

Comment:
I'm ccurious why ci is running on windows and release on linux.

Reply:
it's an excellent question... I'm not entirely sure why. let me look :)

Reply:
it's using ubuntu now :)